LCM/scripts/OmsConfigHostHelpers.py

```diff
@@ -31,7 +31,7 @@ def write_omsconfig_host_telemetry(message, pathToCurrentScript='', level = 'INF
 
     msg_template = '<OMSCONFIGLOG>[%s] [%d] [%s] [%d] [%s:%d] %s</OMSCONFIGLOG>'
     timestamp = datetime.datetime.fromtimestamp(time.time()).strftime('%Y/%m/%d %H:%M:%S')
-    msg_buffer = msg_template % (timestamp, os.getpid(), level, 0, pathToCurrentScript, 0, message)
+    msg_buffer = msg_template % (timestamp, os.getpid(), level, 0, pathToCurrentScript, 0, message.replace('"', '\\"'))
```

Comment:
Should it be '\"' ?

Reply:
Nope. You need to escape the backslash in order to inject a backslash.

Reply:
Can you use a better message encoding or JSON.dumps(message), which will do better handling weird characters (all utf-8, etc). We do that while generating our message.